### ISSUE TITLE
content: ARIA in HTML aria-label Do example seems to be problematic

### ISSUE THREAD

**Issue by BogdanCerovac** [GoogleChrome/web.dev#8795]
`<a aria-label="Some awesome article title">Read More</a>` seems to be problematic for some users and we should recommend better practice (if not directly an WCAG 2.5.3 Label in Name fail).

I would suggest starting with "Read more" also in aria-label, for example: `<a aria-label="Read more about some awesome article title">Read More</a>`.

References;

- [Knowbility WCAG 2.5.3](https://knowbility.org/blog/2018/WCAG21-253LabelInName/#example)
- [WCAG, Understanding Success Criterion 2.5.3: Label in Name](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html#examples)
- [Sara Soueidan, Accessible Text Labels For All](https://www.sarasoueidan.com/blog/accessible-text-labels/#improving-the-experience-for-screen-reader-users-whilst-keeping-it-accessible-to-speech-input-users)

### What page(s) need to be updated?

https://web.dev/learn/accessibility/aria-html/

### Why is this update needed?

`<a aria-label="Some awesome article title">Read More</a>` seems to be problematic for some users (for example screen-reader users that can see the screen and voice control users).

### What's the deadline?

As soon as possible I guess.
